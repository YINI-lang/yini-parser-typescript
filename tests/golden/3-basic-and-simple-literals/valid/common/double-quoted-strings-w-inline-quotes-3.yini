// NOTE: Raw string doesn't support escapes, escape sequences will appear as-is!

^ StringsWithQuotes
double6 = "Hello \'World\'"
