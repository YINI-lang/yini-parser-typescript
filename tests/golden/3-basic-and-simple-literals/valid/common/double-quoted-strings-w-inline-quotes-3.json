{
    "StringsWithQuotes": {
        "double6": "Hello \\'World\\'"
    }
}
